examples/web-chat/src/utils.ts

```diff
@@ -13,3 +14,35 @@ export async function handleCatch(
     return undefined;
   }
 }
+
+export async function saveMultiaddrsToLocalStorage(
```

Comment:
I think it should be integrated into `js-waku`s core code. 

@waku-org/js-waku-developers wdyt?